internal/command/release_create.go

```diff
@@ -0,0 +1,181 @@
+package command
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/simplesurance/baur/v3/internal/command/term"
+	"github.com/simplesurance/baur/v3/internal/log"
+	"github.com/simplesurance/baur/v3/pkg/baur"
+	"github.com/simplesurance/baur/v3/pkg/storage"
+
+	"github.com/spf13/cobra"
+)
+
+const releaseCreateExample = `
+baur release create stable		      create a release named 'stable',
+					      including all tasks
+baur release create --include '*.build' v3    create a release named 'v3',
+					      including only tasks called
+					      'build'`
+
+var releaseCreateLongHelp = fmt.Sprintf(`
+Creates a named snapshot of the status of tasks.
+The snapshot includes information about the tasks and all their outputs.
+
+All tasks included in the release must have a run state of Exist.
+Additional information can be stored with the release database.
+Because it is stored in database, this should not consume too much disk space.
+
+By default all tasks are included in the release. This can be narrowed via the
+--include flag. See %s for more information about the TARGET syntax.
+
+Exit Codes:
+  0 - Success
+  1 - Error
+  %d - Release with the same name already exist
+  %d - One or more tasks are in pending state
```

Comment:
Don't you wish to document the "not exists" exit code here as well?

```
exitCodeAlreadyExist     = 2
exitCodeTaskRunIsPending = 3
exitCodeNotExist         = 4
```

Reply:
`release create` does not exit with `exitCodeNotExist`